component-lib/src/streamlit.ts

```diff
@@ -1,12 +1,11 @@
 /**
- * @license
```

Comment:
Is there any reason why we had these `@license` tags? Probably not relevant.

Reply:
I have found no reason why we should have these annotations. On the other hand, adding this annotation caused problems with the `insert-license` hook, because that meant the header opening marker would have two lines. This I think is a problem that could be solved, but I saw no reason to invest in it if deleting the annotation also solves it.